src/System.Management.Automation/engine/Modules/ImportModuleCommand.cs

```diff
@@ -1968,6 +2039,24 @@ internal override IList<PSModuleInfo> ImportModulesUsingWinCompat(IEnumerable<st
                 return new List<PSModuleInfo>();
             }
 
+            // perform necessary preparations if module has to be imported with NoClobber mode
+            if (filteredModuleNames != null)
+            {
+                foreach(var moduleName in filteredModuleNames)
+                {
+                    PrepareNoClobberWinCompatModuleImport(moduleName, null, ref importModuleOptions);
```

Comment:
Calling like this, where one of two parameters is always null, suggests that there should be two overloads to me

Reply:
I agree, but this syntax historically was used in `ipmo` code, so I'll leave it for now.